### PR TITLE
Grammar corrections

### PR DESCRIPTION
**Pull request by triathletefr23**
### Description

Checked the grammar and all the errors with [Grammarly](www.grammarly.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,19 +95,19 @@ const MyAlertComponent => () => (
 
 ## Concept
 
-Each component should be autonomous (html + css + javascript) and extensible.
+Each component should be autonomous (HTML + CSS + JS) and extensible.
 
-## How It Works
+## How Does It Works
```
Comment (youf-olivier) [inline README.md]: How does it work

**Comment by youf-olivier** [comment]
Conflicting files =)

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -6,20 +6,20 @@ updating documentation, submitting pull requests or patches, and other
...
 
 We are committed to making participation in this project a harassment-free
-experience for everyone, regardless of level of experience, gender, gender
+experience for everyone, regardless of the experience level, gender, gender
 identity and expression, sexual orientation, disability, personal appearance,
-body size, race, age, or religion.
+body size, race, age or religion.
 
-Examples of unacceptable behavior by participants include the use of sexual
+Examples of unacceptable behaviour by participants include the use of sexual
```
[the remark this answers is not in the document]
Comment (guillaumechervetaxa) [reply CODE_OF_CONDUCT.md]: https://www.google.fr/search?hl=fr&q=behavior is better than behaviour ^^

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -6,20 +6,20 @@ updating documentation, submitting pull requests or patches, and other
...
-or private harassment, insults, or other unprofessional conduct.
+or private harassment, insults or other unprofessional conduct.
 
 Project maintainers have the right and responsibility to remove, edit, or
 reject comments, commits, code, wiki edits, issues, and other contributions
 that are not aligned to this Code of Conduct. Project maintainers who do not
 follow the Code of Conduct may be removed from the project team.
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
+Instances of abusive, harassing or otherwise unacceptable behaviour may be
```
[the remark this answers is not in the document]
Comment (guillaumechervetaxa) [reply CODE_OF_CONDUCT.md]: https://www.google.fr/search?hl=fr&q=behavior is better than behaviour ^^

File: README.md [hunk 4]
```diff
@@ -119,5 +118,5 @@ Each component should be autonomous (html + css + javascript) and extensible.
 ## Roadmap
 
 - Update general documentation
-- Continu to migrate to TypeScript component by component
-- Continu to fix and clean code. Make it simpler, faster, lighter
+- Migrating to TypeScript all components
+- Fix the issues and clean code. Make it simpler, faster, lighter
```
[the remark this answers is not in the document]
Comment (triathletefr23) [reply README.md]: 👍

**Pull request merged**